Makefile.conf.in

```diff
@@ -6,7 +6,7 @@ CXX=@CXX@
 CXXFLAGS=@CXXFLAGS@ @OPENMP_CXXFLAGS@
 CXXFLAGS_NOOPENMP=@CXXFLAGS@
 CPPFLAGS=@CPPFLAGS@ @DEFS@
-LDFLAGS=
+LDFLAGS=@LDFLAGS@
```

Comment:
The decision to leave this out was to avoid adding non necessary flags in the linking of the main executable plumed. In your specific case, which flags do you need to add for that executable? In any case, I suspect the correction is fine, I will just have to double check, and perhaps only include it in v2.8 to make sure not to break building scripts that might have been based on this inconsistency

Reply:
It turns out this is not really necessary for the build -- I think I used this for the rpath at some point and then forgot to take it out. In principle, I would expect to be able to pass LDFLAGS, but at this point, there is no practical reason.

Reply:
I think the issue with rpath indeed appeared in #807 and was solved already